Source/WebCore/page/Page.h

```diff
@@ -82,6 +82,10 @@
 #include "DeviceOrientationUpdateProvider.h"
 #endif
 
+#if PLATFORM(COCOA)
+#include <wtf/cocoa/WorkgroupInterval.h>
```

Comment:
Why is this needed in Page.h?

Reply:
Good catch, this should go in the cpp file. Will fix, this will likely be in a new PR, though.

Thanks for reviewing!